src/main/java/com/exasol/errorreporting/Quoter.java

```diff
@@ -0,0 +1,31 @@
+package com.exasol.errorreporting;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * This class quotes objects for the use in error messages.
+ */
+class Quoter {
+
+    private Quoter() {
+        // empty on purpose
+    }
+
+    /**
+     * Quote an objects for the use in error messages.
```

Comment:
```suggestion
     * Returns a quoted string representation of the objectToQuote passed.
```

Reply:
```suggestion
     * Return a quoted string representation of the objectToQuote passed.
```

We usualy write Return insted of Return, Get instead of Gets...